src/Microsoft.Identity.Web/AuthorizeForScopesAttribute.cs

```diff
@@ -32,64 +32,60 @@ public class AuthorizeForScopesAttribute : ExceptionFilterAttribute
         /// <summary>
         /// Scopes to request.
         /// </summary>
-        public string[] Scopes { get; set; }
+        public string[]? Scopes { get; set; }
 
         /// <summary>
         /// Key section on the configuration file that holds the scope value.
         /// </summary>
-        public string ScopeKeySection { get; set; }
+        public string? ScopeKeySection { get; set; }
 
         /// <summary>
         /// Handles the MsalUiRequiredException.
         /// </summary>
         /// <param name="context">Context provided by ASP.NET Core.</param>
         public override void OnException(ExceptionContext context)
```

Comment:
I rewrote this method a bit, but all changes are non-breaking (done with analyzers)